### PR TITLE
Prepare package for publishing to CDSSNC org

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Prepare `gcds-components` for publishing in the cdssnc npm org.

Will merge when we are ready to publish.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,18 @@
 {
-  "name": "gcds-components",
-  "version": "0.0.4",
-  "description": "gcds components proof-of-concept",
+  "name": "@cdssnc/gcds-components",
+  "version": "0.0.1",
+  "description": "GCDS web components",
```
Comment (melaniebmn) [inline package.json:4]: Should we maybe also add some author information aka name, email, url?

**Review by melaniebmn — COMMENTED** [review]
One small suggestion but feel free to merge.

**Review by melaniebmn — APPROVED** [review]
LGTM

**Comment by ethanWallace** [comment]
I want to add a little more to the readme, but after that I think it will be good to merge in.

**Pull request closed without merging**